### DISCUSSION TITLE
[Question] Disable the animation when switching views?

### DISCUSSION THREAD

**Discussion by a0w89fwa0f** [flet-dev/flet#1326] [Q&A] [answered]
### Question

Is there any way to disable the animation when switching views using this for example [routing example](https://gallery.flet.dev/simple-routing/#/). For example when switching views you can see the previous view's content while switching to a different view is it possible to disable/bypass this in some way?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Here is the list of supported transitions: https://flet.dev/docs/controls/page#navigation-transitions

Do you want like 0 delay switching between views?

**Reply by a0w89fwa0f** [reply]
> Do you want like 0 delay switching between views?

Yeah I guess that's what I'm looking for cause even when experimenting with the supported transitions you mentioned I'm not really seeing a difference unless I'm doing something wrong. The best way I can describe it is when switching views I want a way to not see the previous content so as you mentioned a 0 delay switching between views or something.

![Screenshot Example](https://user-images.githubusercontent.com/100470917/234084180-ffb48bca-9281-4bfd-bdef-6499d8c46eb7.png)

Video Example:
https://user-images.githubusercontent.com/100470917/234084451-db0e5d14-cbca-4b3b-bcbb-2de94a2aa312.mp4

**Reply by FeodorFitsner** [reply]
How do you set page's theme? Make sure you are changing dark theme as well.

**Reply by a0w89fwa0f** [reply]
So changing the Dark Theme does allow me to change the transitions, thank you. But they all look mostly the same besides **CUPERTINO**. I'm assuming there is no way to just use the transition or something but make it instant as you suggested earlier e.g. 0 delay switching between views?

**Comment by FeodorFitsner** [comment, marked as answer]
Oh, could you try setting `PageTransitionTheme.NONE`? Looks like the docs are missing that!

**Reply by a0w89fwa0f** [reply]
Works great! Thank you so much I really appreciate it.

**Reply by FeodorFitsner** [reply]
I'm updating docs as we speak... :)

**Comment by insanebaba** [comment]
how to set this in 1.0 Beta?

**Reply by ndonkoHenri** [reply]
```python
    # or page.dark_theme
    page.theme = ft.Theme(
        page_transitions=ft.PageTransitionsTheme(
            android=ft.PageTransitionTheme.FADE_UPWARDS,
            ios=ft.PageTransitionTheme.CUPERTINO,
            linux=ft.PageTransitionTheme.ZOOM,
            macos=ft.PageTransitionTheme.ZOOM,
            windows=ft.PageTransitionTheme.ZOOM,
        )
    )
```
